core/src/com/group66/game/screens/SplitGameScreen.java

```diff
@@ -38,18 +39,21 @@
     private InputHandler inputHandler = new InputHandler();
 
     /** The ball manager. */
-    private BallManager ballManager1 = new BallManager(0);
-    private BallManager ballManager2 = new BallManager(2);
-    private BallManager ballManager3 = new BallManager(1);
+    private BallManager ballManager1;
+    private BallManager ballManager2;
+    private BallManager ballManager3;
     
     /**
      * Instantiates the game screen.
      *
      * @param randomLevel
      *            determines if a set level or a random level is used
      */
-    public SplitGameScreen(Boolean randomLevel) {
+    public SplitGameScreen(Boolean randomLevel, DynamicSettings dynamicSettings) {
```

Comment:
Damn I just relied on check style to find those, but aperantly it only warns you when you have either no javadoc or an extra param (so I assumed it would also check a missing one). O well, merging is fun. Will fix tomorrow morning